src/universal-dapp.js

```diff
@@ -336,7 +421,7 @@ UniversalDApp.prototype.getInstanceInterface = function (contract, address, $tar
       var eventFilter = self.web3.eth.contract(abi).at(address).allEvents()
       eventFilter.watch(parseLogs)
     }
-    $instance.append($title)
+    $instance.get(0).appendChild(title)
```

Comment:
Is it possible to give this inner child a name or an id so that we do not have to change this line if we change the html layout in the future?